README.md

```diff
@@ -158,7 +158,7 @@ The pointcloud, if enabled, will be built based on the aligned_depth_to_color im
    - ```decimation``` - reduces depth scene complexity.
 - **enable_sync**: gathers closest frames of different sensors, infra red, color and depth, to be sent with the same timetag. This happens automatically when such filters as pointcloud are enabled.
 - ***<stream_type>*_width**, ***<stream_type>*_height**, ***<stream_type>*_fps**: <stream_type> can be any of *infra, color, fisheye, depth, gyro, accel, pose, confidence*. Sets the required format of the device. If the specified combination of parameters is not available by the device, the stream will be replaced with the default for that stream. Setting a value to 0, will choose the first format in the inner list. (i.e. consistent between runs but not defined).</br>*Note: for gyro accel and pose, only _fps option is meaningful.
-- ***<stream_type>*_qos**: <stream_type> can be any of *infra, color, fisheye, depth, gyro, accel, pose, confidence*. Sets the QoS by which the topic is published. Available values are the following strings: SYSTEM_DEFAULT, PARAMETER_EVENTS, SERVICES_DEFAULT, PARAMETERS, DEFAULT, SENSOR_DATA.
+- ***<stream_type>*_qos**: <stream_type> can be any of *infra, color, fisheye, depth, gyro, accel, pose, confidence, pointcloud, gyro, accel, pose, imu_info, infra_info, color_info,fisheye_info, depth_info, gyro_info, accel_info, pose_info, fisheye_extrinsics, color_extrinsics, infra_extrinsics*. Sets the QoS by which the topic is published. Available values are the following strings: SYSTEM_DEFAULT, PARAMETER_EVENTS, SERVICES_DEFAULT, PARAMETERS, DEFAULT, SENSOR_DATA.
```

Comment:
Do you think we can avoid mentioning all the topics and just say that there is a parameter for <stream_type>_ros, <stream_type>_info_ros and <stream_type>_extrinsics?
Also, *HID_DEFAULT* and *EXTRINSICS_DEFAULT* are missing. We should inform the users what are their characteristics:
EXTRINSICS_DEFAULT
{
    RMW_QOS_POLICY_HISTORY_KEEP_LAST,
    1,
    RMW_QOS_POLICY_RELIABILITY_RELIABLE,
    RMW_QOS_POLICY_DURABILITY_TRANSIENT_LOCAL,
    RMW_QOS_DEADLINE_DEFAULT,
    RMW_QOS_LIFESPAN_DEFAULT,
    RMW_QOS_POLICY_LIVELINESS_SYSTEM_DEFAULT,
    RMW_QOS_LIVELINESS_LEASE_DURATION_DEFAULT,
    false
};
HID_DEFAULT = DEFAULT with depth of 100.

Reply:
I fixed this [here](https://github.com/IntelRealSense/realsense-ros/pull/1833/commits/c59faa7a5a5a62e21b747b7f7f73f633f9e838b8)
Instead of giving all the details for `EXTRINSICS_DEFAULT` (because there is no detail for all the rest of default QoS), the description is just:
*EXTRINSICS_DEFAULT (= DEFAULT with depth of 1 and transient local durability)*.
 Is it ok for you?

Reply:
It looks very good to me. Thanks.